components/molecules/ComponentDateFilter/component-date-filter.tsx

```diff
@@ -22,15 +24,24 @@ const ComponentDateFilter = ({ setRangeFilter, defaultRange }: ComponentDateFilt
   return (
     <div className="flex items-center text-sm bg-white rounded-lg shrink-0 w-max">
       {dates.map((range, index) => (
-        <div
-          onClick={() => handleFilterClick(range)}
-          className={`px-4 py-1.5 rounded-lg cursor-pointer transition text-light-slate-9 ${
-            activeFilter === range && "border text-light-slate-12 bg-light-slate-5"
-          }`}
+        <Tooltip
           key={index}
+          content={`
+          ${rangeFormatter(range)}
+          ${range === 7 ? "from today" : "from the last commit"}
+        `}
+          direction="top"
         >
-          {rangeFormatter(range)}
-        </div>
+          <div
+            onClick={() => handleFilterClick(range)}
+            className={`px-4 py-1.5 rounded-lg cursor-pointer transition text-light-slate-9 ${
+              activeFilter === range && "border text-light-slate-12 bg-light-slate-5"
+            }`}
+            key={index}
+          >
+            {rangeFormatter(range)}
+          </div>
```

Comment:
Refactor (blocking): this needs to be a button. It doesn't have to be our button component. It can be just a regular `<button />`. It will also need a label. You can wrap it with a label element and add a span inside for screen readers only, e.g. `<span class name="sr-only">some label text</span>`

```suggestion
          <button
            onClick={() => handleFilterClick(range)}
            className={`px-4 py-1.5 rounded-lg cursor-pointer transition text-light-slate-9 ${
              activeFilter === range && "border text-light-slate-12 bg-light-slate-5"
            }`}
            key={index}
          >
            {rangeFormatter(range)}
          </button>
```